tracing-opentelemetry-annotation/build.gradle

```diff
@@ -5,18 +5,12 @@ plugins {
 }
 
 dependencies {
-    api platform (libs.boms.opentelemetry)
-    api platform (libs.boms.opentelemetry.alpha)
-    api platform (libs.boms.opentelemetry.instrumentation)
-    api platform (libs.boms.opentelemetry.instrumentation.alpha)
-    api mn.micronaut.core.processor
-    api projects.micronautTracingAnnotation
-    api libs.opentelemetry.instrumentation.annotations
+    implementation(platform(libs.boms.opentelemetry.instrumentation))
+    implementation(libs.opentelemetry.instrumentation.annotations)
```

Comment:
should not this be api?

Reply:
`tracing-opentelemetry-annotation` only contains `annotationProcessor` classes, it doesn't contain any annotations itself (despite the name 🙄)

I don't *think* we need to pull those in as `api` dependencies 🤔